doc/developer-guide/upgrading.rst

```diff
@@ -370,11 +370,11 @@ The following include files have been renamed:
 HTTPS support
 ^^^^^^^^^^^^^
 
-HTTPS support was moved from the core into a new module, :ref:`mod_ssl`.
+HTTPS support was moved from the core into a new module, *mod_ssl*.
```

Comment:
Ah, sorry, now I see: you just removed the doc link. Just leave this then. A changelog is supposed to be immutable. :wink: We can always create a separate `upgrading-1.0.rst`.